### DISCUSSION TITLE
Index column html content

### DISCUSSION THREAD

**Discussion by MDylan** [area17/twill#2075] [Q&A] [answered]
Hi!

I'm new with Twill.
Is there any way to get html content in the index columns?
Example:
Product module > Field: featured (checkbox)
If I check this, I want to display in the admin/Products/index page the new column with html <div> tag.
It's work with Presenters, but when I try to return a view, it show only {} tag.

![image](https://user-images.githubusercontent.com/2527132/213440444-dbe4e8ee-970d-46f2-a813-afcd27489d53.png)

```
class ProductPresenter extends Presenter {
    public function getFuturedCheck() {
        // return ($this->featured == 1) ? 'Yes' : 'No';
        return view('admin.products.featured', [
            'featured' => $this->featured
        ]);
    }
}

```

![image](https://user-images.githubusercontent.com/2527132/213440687-d261229c-f77e-4dea-91b6-07aa2b458110.png)

**Comment by MDylan** [comment, marked as answer]
Ok, I figured out :)

```
class ProductPresenter extends Presenter {
    public function getFuturedCheck() {
        // return ($this->featured == 1) ? 'Yes' : 'No';
        $view = view('admin.products.featured', [
            'featured' => $this->featured
        ]);
        return htmlspecialchars_decode($view);
    }
}
```
![image](https://user-images.githubusercontent.com/2527132/213443005-199a9966-247d-4995-9a90-4099d6cca758.png)

**Comment by haringsrob** [comment]
Just for a reference, in Twill 3 you can use the table builder to do this:

https://github.com/area17/twill/blob/3.x/docs/content/1_documentation/3_modules/9_tables.md#other-methods
